content/html/concepts/attributes/terms/src/src.md

```diff
@@ -0,0 +1,57 @@
+---
+Title: 'src'
+Description: 'An HTML attribute used to specify the location of a digital resource.'
+Subjects:
+  - 'Web Development'
+  - 'Web Design'
+Tags:
+  - 'Attributes'
+  - 'Elements'
+  - 'Style'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The **`src`** attribute is used to specify the location of a digital source, which is typically a URL. It is required for HTML elements like [`<img>`](https://www.codecademy.com/resources/docs/html/elements/img) so that they can appear on a website or application.
+
+## Syntax
+
+```pseudo
+<element  src="url">
+```
+
+`src` can also be used in the following elements :
+
+| Direction                                                                    | Description                                                                       |
+| ---------------------------------------------------------------------------- | --------------------------------------------------------------------------------- | --- |
+| [`<audio>`](https://www.codecademy.com/resources/docs/html/elements/audio)   | Embeds sound files on websites and applications                                   |     |
+| [`<img>`](https://www.codecademy.com/resources/docs/html/images)             | Embeds online photos, logos, and other pictures into a HTML file.                 |
+| [`<input>`](https://www.codecademy.com/resources/docs/html/elements/input)   | Creates interactive textboxes for online forms.                                   |     |
+| [`<script>`](https://www.codecademy.com/resources/docs/html/elements/script) | Embeds Javascript and other executive code or data into a HTML file .             |
+| `<track>`                                                                    | Specifies the subtitles and closed captions for `<audio>` and `<video>` elements. |
+| [`<video>`](https://www.codecademy.com/resources/docs/html/elements/video)   | Embeds movie clips or other video sources into an HTML file.                      |
+
+## Example 1: Image
+
+The snippet below shows a logo being inserted into an HTML file:
+
+```html
+<img src="logo.png" alt="Codecademy logo" />
+```
+
+> **Note:** When using online images or images from the folders in an IDE workspace, always add `alt` text at the end of the `<img>` element just in case the browser has trouble finding them.
+
+## Example 2: Video
```

Comment:
I believe YouTube and Vimeo use an embed code that uses <iframe> in lieu of <video>. Perhaps we should change this example to better represent a real-world use case.